pom.xml

```diff
@@ -12,8 +12,8 @@
 	<inceptionYear>2015</inceptionYear>
 
 	<properties>
-		<lastaflute.version>1.1.4</lastaflute.version> <!-- about one year old (from now) -->
-		<httpclient.version>4.5.12</httpclient.version>
+		<lastaflute.version>1.2.0</lastaflute.version> <!-- about one year old (from now) -->
```

Comment:
バージョンを上げました

Reply:
Sorry, please revert it.
Because the sub libraries of LastaFlute (e.g. RemoteApi) uses about one year old LastaFlute version for compatibility.
(When it cannot upgrade LastaFlute but it wants to upgrade sub libraries)

Reply:
@jflute 
reverted to lastaflute 1.1.4